cmd/juju/status/status.go

```diff
@@ -384,13 +387,17 @@ func (c *statusCommand) runStatus(ctx *cmd.Context) error {
 		ctx.Infof("Nothing matched specified filter%v.", plural())
 	}
 
-	return nil
+	return numLines, nil
 }
 
 func (c *statusCommand) Run(ctx *cmd.Context) error {
 	defer c.close()
 
-	err := c.runStatus(ctx)
+	if c.watch != 0 {
+		fmt.Printf("\u001Bc")
```

Comment:
Does this work for windows?

Reply:
This is is valid for ANSI compatible terminals but it has to be tested on Windows.

Reply:
I have added additional helpers just for the windows case based on https://github.com/inancgumus/screen/blob/master/clear_windows.go. Anyway, this has to be tested.